content/pt-br/docs/reference/glossary/eviction.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Evicção
+id: eviction
+date: 2022-03-05
+full_link: /pt-br/docs/concepts/scheduling-eviction/
+short_description: >
+    Processo de encerramento de um ou mais Pods em Nós
+aka:
+tags:
+- operation
+---
+
+Evicção é o processo de encerramento de um ou mais Pods em Nós.
+
+<!--more-->
+Existem dois tipos de evicção:
+* [Evicção por pressão no nó](/docs/concepts/scheduling-eviction/node-pressure-eviction/)
+* [Evicção iniciada pela API](/docs/concepts/scheduling-eviction/api-eviction/)

```

Comment:
These pages are not currently translated and I am unsure whether I should translate their descriptions in these links or just keep the page name in English until they get translated.

Reply:
Hmm, I agree with translating the descriptions even the pages aren't translated, today we don't have a clear process to track the pages translations so at the least the descriptions will be in Portuguese.